app/helpers/paymentsHistory.ts

```diff
@@ -0,0 +1,52 @@
+import { Server, ServerApi } from "stellar-sdk";
+import { IPaymentHistory } from "@component/interfaces/payments";
+
+const server: Server = new Server(process.env.NEXT_PUBLIC_SERVER_URL || "");
+const historyLimit: number = 100;
+
+const getPaymentsHistory = async (publicKey: string) => {
+  const accountTransactions = await server
+    .operations()
+    .forAccount(publicKey)
+    .order("desc")
+    .limit(historyLimit)
+    .call();
+
+  return formatPaymentsHistory(accountTransactions);
+};
+
+const formatPaymentsHistory: (
+  unformattedPaymentsHistory: ServerApi.CollectionPage<ServerApi.OperationRecord>
+) => IPaymentHistory[] = (unformattedPaymentsHistory) => {
+  const formattedPaymentsHistory: IPaymentHistory[] = [];
+
+  unformattedPaymentsHistory.records.forEach(
+    (payment: ServerApi.OperationRecord) => {
```

Comment:
I think you can destructure the payment properties here.

Reply:
I have destructured the 'payment' argument.